pkg/sysinfo/nummem.go

```diff
@@ -0,0 +1,31 @@
+// +build amd64, arm
```

Comment:
Is there any reason we need this - is it better than the +build linux version below? I'm leery about maintaining parallel implementations if we don't have to

Reply:
Yeah @mheon I guess you are right, I was originally doing that one since arm and amd64 supported a more direct NUMA function but the linux implementation works for them as well.

Reply:
@mheon pushed and removed, can add back later if we realize we want to support it.